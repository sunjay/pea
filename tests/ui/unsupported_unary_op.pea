fn main() {
    println!(!b"abc");
}
