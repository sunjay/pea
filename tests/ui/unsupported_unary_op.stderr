[tests/ui/unsupported_unary_op.pea:2:15-2:20] error: mismatched types: expected `bool`, found: `[u8]`

error: aborting due to 1 previous error

